slides/content/02-kotlin/_index.md

```diff
@@ -1206,10 +1203,9 @@ Kotlin's type system supports generics
 * No higher kinded types (they are in Scala)
 * No type lambdas (they are in Scala)
 * Declaration-site variance (absent in Java)
-* Generic type reification via inlining (not found in Java nor Scala)
+* Generic type reification via inlining (not found in Java, kind of Scala inlining)
 ```scala
-trait Functor[F[_]] // There is no Kotlin equivalent for these lines
-type MapFunctor = Functor[({ type T[A] = Map[Int, A] })#T]
+type MapFunctor = [A] =>> Map[Int, A] // There is no Kotlin equivalent for this line
```

Comment:
Scala 3.x has a "human" readable syntax for type lambda (probably we do not introduce it during the course, but is also the previous example was quite esoteric for the one that attend PPS I think.